ci/docker/Dockerfile.maldet.el.8

```diff
@@ -0,0 +1,47 @@
+# Copyright 2018-2022 Intel Corporation
```

Comment:
If you are not going to use Docker any more, this isn't necessary then, is it?

Reply:
Of course we are going to be using docker.